src/app/pages/signing/signing.component.html

```diff
@@ -73,4 +85,4 @@ <h5 class="modal-title" id="submitConfirmModalTitle">Are you sure you want to su
       </div>
     </div>
   </div>
-</div>
+</div>
```

Comment:
No newline at end of file
```suggestion
</div>

```